src/aioauth/utils.py

```diff
@@ -41,20 +41,20 @@ def get_authorization_scheme_param(
     return scheme, param
 
 
-def list_to_scope(scope: Optional[List] = None) -> Text:
-    """Convert a list of scopes to a space separated string."""
+def list_to_str(scope: Optional[List] = None) -> Text:
+    """Convert a list of string to a space separated string."""
     if isinstance(scope, str) or scope is None:
```

Comment:
This looks to me like defensive programming. If the `scope` is `Optional[List]`, mypy will lint when it's not used properly. This also removes the need to raise a ValueError. The function should check for `None` and return an empty string, or return the joined strings.